x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/column_data.tsx

```diff
@@ -4,79 +4,101 @@
+    {
+      id: ACTUAL_CLASS_ID,
+      display: <span />,
+    },
+  ];
 
-  confusionMatrixData.forEach((classData: any) => {
-    const correctlyPredictedClass = classData.predicted_classes.find(
-      (pc: PredictedClass) => pc.predicted_class === classData.actual_class
-    );
-    const incorrectlyPredictedClass = classData.predicted_classes.find(
-      (pc: PredictedClass) => pc.predicted_class !== classData.actual_class
-    );
+  let showOther = false;
 
-    let accuracy;
-    if (correctlyPredictedClass !== undefined) {
-      accuracy = correctlyPredictedClass.count / classData.actual_class_doc_count;
-      // round to 2 decimal places without converting to string;
-      accuracy = Math.round(accuracy * 100) / 100;
-    }
+  confusionMatrixData.forEach(classData => {
+    const otherCount = classData.other_predicted_class_doc_count;
 
-    let error;
-    if (incorrectlyPredictedClass !== undefined) {
-      error = incorrectlyPredictedClass.count / classData.actual_class_doc_count;
-      error = Math.round(error * 100) / 100;
+    if (otherCount > 0) {
+      showOther = true;
     }
 
-    let col: any = {
+    const col: any = {
       actual_class: classData.actual_class,
       actual_class_doc_count: classData.actual_class_doc_count,
+      other: otherCount,
     };
 
-    if (correctlyPredictedClass !== undefined) {
-      col = {
-        ...col,
-        predicted_class: correctlyPredictedClass.predicted_class,
-        [correctlyPredictedClass.predicted_class]: accuracy,
-        count: correctlyPredictedClass.count,
-        accuracy,
-      };
-    }
+    const predictedClasses = classData.predicted_classes || [];
 
-    if (incorrectlyPredictedClass !== undefined) {
-      col = {
-        ...col,
-        [incorrectlyPredictedClass.predicted_class]: error,
-        error_count: incorrectlyPredictedClass.count,
-      };
+    columns.push({ id: classData.actual_class });
+
+    for (let i = 0; i < predictedClasses.length; i++) {
+      const predictedClass = predictedClasses[i].predicted_class;
+      const predictedClassCount = predictedClasses[i].count;
+      col[predictedClass] = predictedClassCount;
     }
 
     colData.push(col);
   });
 
-  const columns: any = [
+  if (showOther) {
+    columns.push({ id: OTHER_CLASS_ID });
+  }
+
+  return { columns, columnData: colData };
+}
+
+export function getTrailingControlColumns(numColumns: number, setShowFullColumns: any) {
+  return [
     {
-      id: ACTUAL_CLASS_ID,
-      display: <span />,
+      id: 'actions',
+      width: 60,
+      headerCellRender: () => `${numColumns} more`,
+      rowCellRender: function RowCellRender() {
+        const [isPopoverOpen, setIsPopoverOpen] = useState<boolean>(false);
+        return (
+          <EuiPopover
+            isOpen={isPopoverOpen}
+            anchorPosition="upCenter"
+            button={
+              <EuiButtonIcon
+                aria-label="show actions"
```

Comment:
We should be internationalizing all these aria-labels.

Reply:
Added in 422c7aacdfc9d129a7877af2d9f1c8ebede15f80